example/simple_filter.cpp

```diff
@@ -0,0 +1,38 @@
+//include libraries for image io, kernel definition and convolution
+#include <boost/gil/image.hpp>
+#include <boost/gil/typedefs.hpp>
+#include <boost/gil/extension/io/jpeg.hpp>
+#include <boost/gil/extension/numeric/kernel.hpp>
+#include <boost/gil/extension/numeric/convolve.hpp>
+
+
+int main(int argc,char *argv[]) {
+    using namespace boost::gil;
+    if(argc!=2){
+        printf("Usage: simple_filter <image_path>");
+        return 0;
+    }
```

Comment:
Please use 'std::cerr' rather than `printf`, and return `-1` to indicate the usage error.

Reply:
I'd also allow to use the C standard macros https://en.cppreference.com/w/cpp/utility/program/EXIT_status